<launch>

    <!-- Launch mission_planner_node -->
    <node pkg="mission_planner" type="mission_planner_node" name="mission_planner_ros" output="screen" ns="info_drone_1" />

</launch>
